.github/workflows/test.yml

```diff
@@ -21,14 +21,15 @@ jobs:
   workspace_deluxe_tests:
     runs-on: ubuntu-latest
     strategy:
```

Comment:
There are probably documentation changes that are needed to note that that workspace now requires java 11. In the readme for sure, but search the docs folder to see if there's anywhere else>

Reply:
I think it would be better to split out the docs updates into a separate, followup PR because there are several files involved.

I'd also like to remove a number of files that are no longer used in a followup PR.

Reply:
There are now docs changes in this PR - are there any more to do?

Reply:
Discussed over slack; see also #667 .

Reply:
:+1: